File: VbFormsFlashcards/VbFormsFlashcards/FlashCardsArchivos.vb
Imports System.IO
Public Class FlashCardsArchivos


    Public Sub GuardarTexto(destino As String, pregunta As String, respuesta As String)

        Dim destinofinal As String = destino & pregunta & ".txt"

        File.WriteAllText(destinofinal, respuesta)


    End Sub



End Class

File: CsFormsFlashcards/CsFormsFlashcards/FlashCardsArchivos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.IO;

namespace CsFormsFlashcards
{


    class FlashCardsArchivos
    {
  

        public void Guardartexto(string destino, string pregunta, string respuesta)
        {

            string destinofinal = destino + pregunta + ".txt";

            File.WriteAllText(destinofinal,respuesta);

      
        }

    }
}
